doc/system-upgrade.rst

```diff
@@ -71,6 +71,10 @@ Options
     REQUIRED. The version to upgrade to. Sets ``$releasever`` in all enabled
     repos. Usually a number, or ``rawhide``.
 
+``--downloaddir=<path>``
+    Directory where downloaded packages are stored. Default location is
+    /var/lib/dnf/system-upgrade
```

Comment:
This is not certainly true. The destination is /var/lib/dnf/system-upgrade/<repoid>/packages